README.md

```diff
@@ -23,7 +23,9 @@ This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-opti
 ## API 
 
 For the API we are using the [Firebase REST API](https://firebase.google.com/docs/database/rest/start).  
-For this project, the api can be found under https://devstaff-draw-default-rtdb.europe-west1.firebasedatabase.app
+For this project, the api can be found under BASE_URL 
```

Comment:
The variable should be FIREBASE_URL, so I suggest to create a .env file with empty value.

Reply:
In the README file let's add that someone needs to set the env variable in order to be able to run it locally.